service_container/alias_private.rst

```diff
@@ -170,11 +176,17 @@ or you decided not to maintain it anymore), you can deprecate its definition:
         app.mailer:
             alias: '@App\Mail\PhpMailer'
 
-            # this will display a generic deprecation message...
-            deprecated: true
+            # this outputs the following generic deprecation message:
+            # Since acme/package 1.2: The "app.mailer" service is deprecated. You should stop using it, as it will be removed in the future
+            deprecated:
+                package: 'acme/package'
+                version: '1.2'
 
-            # ...but you can also define a custom deprecation message
-            deprecated: 'The "%alias_id%" alias is deprecated. Do not use it anymore.'
+            # you can also define a custom deprecation message (%sevice_id%/%alias_id% placeholder is available)
```

Comment:
If i'm not wrong, this section explains only how to deprecate aliases not services, the '%service_id%' should be removed from here, only `%alias_id%` could be used when deprecating aliases